locales/locales.php

```diff
@@ -1846,6 +1879,14 @@ public function __construct() {
 		$sw->slug = 'sw';
 		$sw->google_code = 'sw';
 		$sw->facebook_locale = 'sw_KE';
+		
+		$syr = new GP_Locale();
+		$syr->english_name = 'Syriac';
+		$syr->native_name = 'Syriac';
+		$syr->lang_code_iso_639_3 = 'syr';
+		$syr->country_code = 'IQ';
+		$syr->wp_locale = 'syr';
+		$syr->slug = 'syr';
```

Comment:
Facebook has a locale for `Syriac`: sy_SY, not sure if it's the same since SY and IQ.